SamplesDashboard/SamplesDashboard/Services/SampleService.cs

```diff
@@ -136,10 +137,7 @@ public async Task<IEnumerable<DependenciesNode>> GetDependencies(string sampleNa
           
             if (graphQLResponse.Data.Organization.Repository != null)
             {
-                var dependencies =
-                    graphQLResponse.Data.Organization.Repository.DependencyGraphManifests.Nodes.SelectMany(n =>
-                        n.Dependencies.Nodes);
-                return dependencies;
+                return graphQLResponse.Data.Organization.Repository;
```

Comment:
This could be simplified to remove the surrounding if block.
Lets also fix the xml comment of the function to have the client parameter removed